wandb/sdk/wandb_run.py

```diff
@@ -838,6 +838,8 @@ def _display_run(self):
                     click.style(run_url, underline=True, fg="blue"),
                 )
             )
+            if not self._settings.offline:
+                wandb.termlog("Run `wandb off` to turn off syncing.")
```

Comment:
Maybe add `or set WANDB_MODE=offline` to this message?

Reply:
good idea, will add in next pr